### ISSUE TITLE
Improve our PageSpeed Insights score

### ISSUE THREAD

**Issue by Minoru** [Minoru/blog.debiania.in.ua#47]
https://pagespeed.web.dev/ notes a few problems:

- the RSS icon doesn't have alt text and title
- no `lang` attribute on `html` tag
- no meta-description of the site
- links don't have underlines

Plus a few more things.

**Comment by Vikas3100** [comment]
hii @Minoru 
I am Interested in this issue and want to contribute in this project please assign it to me .

**Comment by Minoru** [comment]
Hey there!

This is merely a personal blog, not a full-on project, but I'm excited that you took interest in it!

You can take care of these three points:

> - the RSS icon doesn't have alt text and title
> - no lang attribute on html tag
> - no meta-description of the site

As for underlines for links, I'm not yet sure what style I want. You can send a separate PR if you have ideas, but please don't expect me to merge it.

Feel free to look at other PageSpeed Insights too, and fixing the more technical ones, but again, please send separate PRs for those.

Finally, please note that I don't have a CI, and the site is not automatically deployed. You'll have to build and test locally before submitting a PR, and you won't see the results on pagespeed.web.dev until I manually deploy.
